test/integration/scheduler/extender_test.go

```diff
@@ -305,4 +346,19 @@ func DoTestPodScheduling(ns *v1.Namespace, t *testing.T, cs clientset.Interface)
 		t.Fatalf("Failed to schedule using extender, expected machine3, got %v", myPod.Spec.NodeName)
 	}
 	t.Logf("Scheduled pod using extenders")
+
+	pod.Name = failPodName
+	failPod, err := cs.Core().Pods(ns.Name).Create(pod)
+	if err != nil {
+		t.Fatalf("Failed to create pod: %v", err)
+	}
+
+	time.Sleep(time.Second * 2)
```

Comment:
I will make another pass once the other PR is merged. I wanted to quickly put something together so folks get an idea on how much of a change this is (per the last sig discussion).